### ISSUE TITLE
Baba vs. Zero

### ISSUE THREAD

**Issue by Codiekitty** [ArcadeHustle/CSI-Paprium#27]
![zerovsbaba](https://user-images.githubusercontent.com/93952332/140859959-312bc909-de44-43af-8d8a-bf7f490c7c2d.png)

**Comment by 2blackbar** [comment]
Yup, the purpose of this repo is to feel better about dumping paprium (stealing it) and uploading , asking dev did not made it "right" ,  proving that game used poses from other games for inspiration wont make it right, id change thw whole character of this repo but you do you PoLicEman thief.
Its not like what you do is crap but the way of doing it is just wrong.

**Comment by ArcadeHustle** [comment]
LOL you are a super butt hurt fan boy. Right to repair broken games is a thing.

**Comment by ArcadeHustle** [comment]
@2blackbar please change your avatar. 
![image](https://user-images.githubusercontent.com/41961787/175618087-ddc543ff-fb2d-4648-8ea4-535257dea159.png)
